### DISCUSSION TITLE
切换到英文模式 切换到中文模式 各自设置一个单独的快捷键，而不是 toggle

### DISCUSSION THREAD

**Discussion by chen-bao-x** [rime/home#1101] [Q&A] [answered]
我想设置快捷键：
     `command + :` 输入英文，无论当前是英文模式还是中文模式。
    `command + shift + ;` 输入中文，无论当前是英文模式还是中文模式。

**Comment by lotem** [comment, marked as answer]
可以的

參閱相同問題： https://github.com/rime/home/issues/454

**Reply by chen-bao-x** [reply]
感谢大佬， #454  解决了我的问题。


我又遇到了一个新的问题，
```yaml
- { accept: "⌘+semicolon", set_option: ascii_mode, when: always } #切换到英文输入模式
```
这里的 `⌘` 对应的按键是什么呢？



Meta Meta_L 
Super Super_L
 Hyper Hyper_L
 都不对应 command key `⌘`

**Reply by LEOYoon-Tsaw** [reply]
command用不了吧

**Reply by chen-bao-x** [reply]
> command用不了吧

原来如此。
